test/k8sT/Bandwidth.go

```diff
@@ -51,6 +52,8 @@ var _ = Describe("K8sBandwidthTest", func() {
 
 	BeforeAll(func() {
 		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+		err := kubectl.AddRegistryCredentials(config.CiliumTestConfig.RegistryCredentials, config.RegistryDomain)
```

Comment:
I think this should be moved to either `helpers.CreateKubectl` or `kubectl.PrepareCluster`, I don't think there is need for it to be in every test suite.